coldfront/core/project/tests/test_views/test_new_project_views/test_savio_project_request_wizard.py

```diff
@@ -36,6 +37,27 @@ def test_post_creates_request(self):
         computing_allowance = Resource.objects.get(name=BRCAllowances.FCA)
         allocation_period = get_current_allowance_year_period()
 
+        kwargs = {
```

Comment:
This should be moved to `project.tests.test_forms.test_new_project_forms.test_savio_project_existing_pi_form`.